perf/flame/get_perfdata.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+WD=$(dirname "${0}")
+WD=$(cd "${WD}" && pwd)
+
+FILENAME=${1:?"perffilename"}
+DURATION=${2:?"duration"}
+FREQ=${3:-"99"}
+
+PID=$(pgrep envoy)
+
+# This is specific to the kernel version
+# provided by `linux-tools-generic`
+PERF="/usr/lib/linux-tools/4.4.0-131-generic/perf"
```

Comment:
any ideas on how to make this more portable?   or maybe just an explicit existence check + useful error message?